From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Anton Tayanovskyy <anton@pulumi.com>
Date: Tue, 6 May 2025 15:19:49 -0400
Subject: [PATCH] Add shim package to expose internals


diff --git a/shim/shim.go b/shim/shim.go
new file mode 100644
index 00000000..8c3b3bac
--- /dev/null
+++ b/shim/shim.go
@@ -0,0 +1,12 @@
+package shim
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	p "github.com/mongodb/terraform-provider-mongodbatlas/internal/provider"
+)
+
+func FrameworkProvider() provider.Provider { return p.NewFrameworkProvider() }
+
+func SDKProvider() *schema.Provider { return p.NewSdkV2Provider() }
